### DISCUSSION TITLE
Questions about Paru on Artix

### DISCUSSION THREAD

**Discussion by Longview3k** [Morganamilo/paru#571] [Q&A] [answered]
So I was looking on the front page of Artix and I saw a couple of AUR helpers being removed from the main Artix Repos because these AUR Helpers have a tendency of Removing installed packages from Artix's System Repo and installing packages from Arch's Core Repo causing the system to break.

I wanted to know if Paru depends or installs dependencies from Arch's Main Repositories, or if Paru just uses whatever repos that come with the distro

**Comment by Morganamilo** [comment]
Do you have a source for this? Every helper should be deferring to pacman which will respect the repo order.

**Reply by Longview3k** [reply]
https://forum.artixlinux.org/index.php/topic,3186.0.html

An Artix Dev started this forum on the site

**Comment by Morganamilo** [comment, marked as answer]
That thread just seems to call out pamac. Which I believe as pamac is a bit weird and does weird things. Annoyingly they don't link to an issue though.

I imagine no other aur helpers would have this problem.

**Reply by Longview3k** [reply]
Gotcha, I've never used Pamac so I wasn't sure if it was just Pamac or other AUR Helpers. Thank you
